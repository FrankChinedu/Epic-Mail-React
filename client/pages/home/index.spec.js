import React from 'react';
import { shallow } from 'enzyme';
import { HomeComponent } from './index';

describe('Home', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<HomeComponent debug />);

    expect(component).toMatchSnapshot();
  });
});
