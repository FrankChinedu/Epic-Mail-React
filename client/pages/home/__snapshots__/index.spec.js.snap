// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render correctly in "debug" mode 1`] = `
<Fragment>
  <div
    className="main"
  />
  <div
    className="index-body flex flex-dir-col justify-content-sb"
  >
    <header
      className="nav-header"
    >
      <div
        className="flex justify-content-sb"
      >
        <Link
          to="/"
        >
          <h1
            className="brand-name flex"
          >
            <span
              className="black"
              style={
                Object {
                  "color": "white",
                }
              }
            >
              EPIC Mail
            </span>
            <span
              className="brand-image"
            />
          </h1>
        </Link>
        <div
          className="align-self"
        >
          <Link
            className="li-sty-none txt-white"
            to="/signup"
          >
            <h3
              className="m-0  sign-in-out join-us"
            >
              Join us
            </h3>
          </Link>
        </div>
      </div>
    </header>
    <div
      className="index-content"
    >
      <div
        className="col-5 center-text"
      >
        <h2
          className="index-sub-title"
        >
          Itâ€™s all coming together
        </h2>
        <p
          className="index-sub-cont"
        >
          exchanging messages over the internet just became way easier
        </p>
        <div>
          <Link
            className="li-sty-none txt-white inline-block"
            to="/signin"
          >
            <h3
              className="m-0 join-us width-fit-content"
            >
              Sign In
            </h3>
          </Link>
        </div>
      </div>
    </div>
    <footer
      className="flex justify-content-flex-end footer"
    >
      <a
        href="#"
      >
        <h5
          className="txt-white contact-us"
        >
          contact us
        </h5>
      </a>
    </footer>
  </div>
</Fragment>
`;
